both/collections/users.js

```diff
@@ -0,0 +1,84 @@
+Schema.passwordEdit = new SimpleSchema({
+  old: {
+    type: String,
+    label: "Old Password",
+    min: 6,
+    max: 30
+  },
+  new: {
+    type: String,
+    label: "New Password",
+    min: 6,
+    max: 30
+  },
+  newConfirm: {
+    type: String,
+    label: "Confirm Password",
+    min: 6,
+    max: 30,
+    custom: function() {
+      // This custom function renders an error, if this field is not equal to
+      // the new Password field supplied in the form.
+      if (this.value !== this.field('new').value) {
+        return "passwordMismatch";
+      }
+    }
+  }
+});
+
+Schema.UserProfile = new SimpleSchema({
+  gender: {
+    type: String,
+    allowedValues: [
+      "Female",
```

Comment:
These values need to be internationalized in the UI, but can be left in English here.